src/components/Accordion/Accordion.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import AccordionItem from './AccordionItem';
+
+const CN = 'accordion';
+
+class Accordion extends Component {
+  constructor(props) {
+    super(props);
+
+    this.clickHandler = this.clickHandler.bind(this);
+    this.state = {
+      hide: [],
+      currentIndex: null,
+    };
+  }
+
+  componentDidMount() {
+    const { data, open } = this.props;
+    const hide = data.map((el) => !open);
```

Comment:
Ok, so you can get access to this data directly in constructor. 
I would suggest creating function like dataMapper which will get this original data and returns new array of dataElement (object from data raray) but would add props `isHidden`to them. You can use it in constructor with your props.data. You will be able to handle displaying/not-displaying elements base on ONE data sourse which is more useful in case if your component will grow and the amount of different data that is used inside it will grow too.
And in constructor you can put it to state and operate with this new array instead of 2 arrays: actually your data and a synthetic array of booleans just to handle displaying.

Reply:
Great idea!